### DISCUSSION TITLE
Auswahl Anzahl Karten und Teilnehmer nicht möglich

### DISCUSSION THREAD

**Discussion by ElbPsi** [fabiankachlock/zwoo#259] [Feature Requests]
Hallo Team,
bei den Spiel Einstellungen ist es nicht möglich die Anzahl der Karten und die Anzahl der Teilnehmer einzugeben.

Grüße, Elb

**Comment by fabiankachlock** [comment]
Hallo Elb,
vielen Dank für die Anmerkung.
zwoo ist mit dem Minimum an für das Spielen benötigten Features in die BETA gegangen. Zurzeit arbeiten wir hart daran, weitere essenzielle Features, zu denen auch das Ändern von Regeln/Spiel-Einstellungen zählt, einzubauen.
Allerdings ist das Ändern von Regeln/Spiel-Einstellungen mittlerweile implementiert (#254) und sollte mit dem nächsten Update (voraussichtlich diese Woche) verfügbar sein.

zwoo Team

**Reply by fabiankachlock** [reply]
linked: #263

**Comment by fabiankachlock** [comment]
Lösungsversion: v1.0.0-beta.4
